src/test/java/uk/gov/register/functional/DataDownloadFunctionalTest.java

```diff
@@ -29,7 +29,7 @@
 public class DataDownloadFunctionalTest {
 
     @ClassRule
-    public static RegisterRule register = new RegisterRule("address");
+    public static RegisterRule register = new RegisterRule();
```

Comment:
ditto.

more generally, I have a goal that `RegisterRule` should only ever be in a `@ClassRule` - and that's the consistency I'm aiming for